enterprise/backend/test/metabase_enterprise/semantic_search/repair_test.clj

```diff
@@ -47,13 +47,21 @@
   [gate-contents id]
   (some #(when (= (:id %) id) %) gate-contents))
 
+(defn- clear-gate-table!
+  "Clear all entries from the gate table to ensure clean test state"
+  [pgvector gate-table-name]
+  (jdbc/execute! pgvector
+                 (-> {:delete-from [(keyword gate-table-name)]}
+                     (sql/format :quoted true))))
+
 (deftest repair-index-integration-test
   (testing "repair-index! properly handles document additions and deletions via gate table"
     (mt/with-premium-features #{:semantic-search}
       (semantic.tu/with-index!
         (let [pgvector       semantic.tu/db
               index-metadata semantic.tu/mock-index-metadata
               gate-table     (:gate-table-name index-metadata)
+              _              (clear-gate-table! pgvector gate-table)
```

Comment:
huh, `semantic.tu/with-index!` should be cleaning up the gate table via `open-temp-index-and-metadata!` -> `cleanup-index-metadata!`

Reply:
this is more about the gate table being dirty going into this test. it's possible other tests not using `with-index!` write to it

Reply:
yeah but `open-temp-index-and-metadata!` drops the tables both on open and close, so unless another test is running in parallel (in which case this call to `clear-gate-table!` might not help either), it should start from a clean slate.